.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
+
+php:
+  - 7.0
+  - 7.1
+
+sudo: false
+
+cache:
+  directories:
+    - cache
+    - vendor
+    - $HOME/.composer/cache
+
+install:
+  - npm install
+  - composer install --no-interaction --prefer-source
+
+before_script:
+  - if find . -name "*.php" ! -path "./vendor/*" -exec php -l {} \; | grep "unexpected $end"; then exit 1; fi
+
+script:
+  - echo 'scripts'
```

Comment:
@prathmeshranaut @npari  PHP unit module has not been pushed yet. So if we replace it with the command Travis will show error. The purpose of this is to test that travis has successfully reached to script-section in the logs. If you still suggest I can remove it directly.